pom.xml

```diff
@@ -811,5 +732,103 @@
         <debugJavaOptions>-Xdebug -Xrunjdwp:server=y,transport=dt_socket,address=${debugJavaPort},suspend=n</debugJavaOptions>
       </properties>
     </profile>
+    <profile>
```

Comment:
The tests are failing because it hasn't started Docker yet. My guess is that the profile is introducing non-determinism in the plugin order. You'll need to rework this as discussed.

Reply:
Fixed